### PR TITLE
fix: menu icon & breadcrumbs

### PR DESCRIPTION
**Pull request by Keshraf**
## Description

1. Fix Breadcrumbs to make it more visible
2. Fix the Menu Icon in the Projects Page to make it black on Description page, and white on rest

## Updates

[Please enter a brief summary of updates to this PR during code review - if any]

## Checklist

- [x] Ran `npm run lint:fix` to format code
- [ ] Requested at least one reviewer
- [x] Connected PR to Trello ticket (steps below)
- [ ] Addressed code review comments
- [ ] Gained at least one approval for your PR

## Connecting your PR to the corresponding ticket:
- Click on the "GitHub" button found on the right of your Trello ticket
- Choose "Attach PR" from the dropdown list
- Link your GitHub account (if you haven't done so already)
- Navigate to`VCL-content-platform` and choose your PR from the dropdown.

### CODE REVIEW

**Comment by Keshraf** [comment]
[add contrast and enlarge font size for get involve page breadcrumb and expansion icon](https://trello.com/c/xID7FZqr/193-add-contrast-and-enlarge-font-size-for-get-involve-page-breadcrumb-and-expansion-icon)

File: package-lock.json
Comment (ElisaHu1) [inline package-lock.json:1]: please revert this back if you are not changing anything in package.json

**Comment by ryanyae** [comment]
@ElisaHu1 I wonder if it will be more beneficial if we were to add a black outlining to the text in the breadcrumbs in-order to make it just a tad bit more visible?

**Review by ryanyae — CHANGES_REQUESTED** [review]
Besides the question that I brought up to Elisa, just address Elisa's comment about the packlock.json file. Thank you for your hard work Ketan!

**Comment by ElisaHu1** [comment]
> @ElisaHu1 I wonder if it will be more beneficial if we were to add a black outlining to the text in the breadcrumbs in-order to make it just a tad bit more visible?

i agree! Ketan can you adjust this as well?

**Comment by Keshraf** [comment]
@ryanyae @ElisaHu1 

I added the text border to the text, but it's not looking good. And I have removed pkg-lock

Screenshot 2023-10-26 at 5 31 39 PM
Screenshot 2023-10-26 at 5 31 48 PM

**Comment by ElisaHu1** [comment]
> I added the text border to the text
where did you add it? I only see a commit for rm lock file

**Comment by ryanyae** [comment]
@Keshraf I do agree that this looks pretty bad. Wonder if there is another way of making this more visible without changing the colour? Wonder if the opacity has to to anything with it.

**Comment by Keshraf** [comment]
> @Keshraf I do agree that this looks pretty bad. Wonder if there is another way of making this more visible without changing the colour? Wonder if the opacity has to to anything with it.

Opacity won't affect it much. I have already swapped the colors according to the page theme. To make it even more visible - we can increase size further (but it will too large), or we can choose different color (that needs input of the design team right?)

**Review by ElisaHu1 — APPROVED** [review]

**Review by ryanyae — CHANGES_REQUESTED** [review]
Sorrry for the very late review, but the ProjectJoin's breadcrumb looks very inconsistent with the other breadcrumbs, 

image

image
image

**Comment by Keshraf** [comment]
@ryanyae, the colour was changed according to the page theme because even if I made the breadcrumbs bold, it would still not be visible on some pages. I feel that this is a design issue rather than a dev issue, and an input from the design team would help a lot.

**Comment by ryanyae** [comment]
> @ryanyae, the colour was changed according to the page theme because even if I made the breadcrumbs bold, it would still not be visible on some pages. I feel that this is a design issue rather than a dev issue, and an input from the design team would help a lot.

Sorry I should have specified that the marginning is off.

**Comment by Keshraf** [comment]
@ryanyae can you be more specific? The spacing used in the breadcrumb is the same on all the pages. If you see that the breadcrumbs' position differs on each page, that's because each page has a different layout; hence, the breadcrumbs are shifted. In terms of colour, if I keep the same colour on all the pages, then the breadcrumbs won't be visible. Ideally, the design team should suggest a change here. They should specify a colour that works for all the pages. Because I have already made the breadcrumb bold and larger, but it's still not making much difference. 

Let me know your thoughts on this. There are two problems here:
- inconsistent position due to individual page's different layout (here we have to make all the pages consistent to fix it)
- inconsistent color of the breadcrumbs (if we keep the same color on both the dark and light page, then the breadcrumbs wont be visible on at least one of them.)

If you have any other ideas to fix these issues, then please let me know. We can discuss this on slack as well.

**Comment by ryanyae** [comment]
> @ryanyae can you be more specific? The spacing used in the breadcrumb is the same on all the pages. If you see that the breadcrumbs' position differs on each page, that's because each page has a different layout; hence, the breadcrumbs are shifted. In terms of colour, if I keep the same colour on all the pages, then the breadcrumbs won't be visible. Ideally, the design team should suggest a change here. They should specify a colour that works for all the pages. Because I have already made the breadcrumb bold and larger, but it's still not making much difference.
> 
> Let me know your thoughts on this. There are two problems here:
> 
> * inconsistent position due to individual page's different layout (here we have to make all the pages consistent to fix it)
> * inconsistent color of the breadcrumbs (if we keep the same color on both the dark and light page, then the breadcrumbs wont be visible on at least one of them.)
> 
> If you have any other ideas to fix these issues, then please let me know. We can discuss this on slack as well.

I was pointing out that we should fix the breadcrumb's position on the "Join Our Team" page, this page is inconsistent with all the others. I agree on the colour changes and bolding, I think it is fine how it is at the moment. Do you still think that this position change is unnecessary, or even unpractical? If so we can just drop this conversation and I will just merge in the changes.

**Comment by Keshraf** [comment]
> I was pointing out that we should fix the breadcrumb's position on the "Join Our Team" page, this page is inconsistent with all the others. I agree on the colour changes and bolding, I think it is fine how it is at the moment. Do you still think that this position change is unnecessary, or even unpractical? If so we can just drop this conversation and I will just merge in the changes.

No that makes sense, I will fix the page layout for you. Make it more aligned with the other pages.

**Review by ryanyae — APPROVED** [review]
Looks good to me!

**Pull request merged**